### ISSUE TITLE
Material Theme Side Navigation Screen-reader Fixes

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#4226]
From screenreader testing (https://github.com/BirkbeckCTP/janeway/issues/4194), looking at the narrow screen navigation the following issues were identified:

- [x] Sample 2, item 2. home, link, list start - but nothing visible on the page and is a duplicate of the link above.
- [x] Sample 2, item 3. Visually, at phone screen width there is a hamburger menu, but to the screen reader, no such thing exists, but instead the whole uncollapsed menu is read out despite being hidden. These menu items are not identified as a menu or as navigation.
- [x] Sample 2, item 4. items in the menu (articles, submissions, journal policies, about) are read followed by "arrow underscore drop underscore down" and identified as links. `<i class="material-icons right">arrow_drop_down</i>`
- [x] Sample 2, item 8. "Menu, link, end, navigation, end, banner". This is the hamburger menu, and when activated the menu does pop up, but you can't navigate the list, as you already have - what pops up visually is the list that I've just described above, but it has already been navigated, when advancing one reaches the article title instead. Further, there is no way now to dismiss this menu without refreshing the whole page, activating the menu link again does nothing.

**Comment by StephDriver** [comment]
these specific issues have been addressed, however it should be noted that the Material navigation is not fully screen-reader accessible and further work on this has been halted in favour of implementing an accessibility mode and signposting screen reader users towards it.
